plugins/modules/autoscaling_group.py

```diff
@@ -341,13 +373,14 @@
 """
 
 EXAMPLES = r"""
-# Basic configuration with Launch Configuration
+# Basic configuration with Launch Template
 
 - amazon.aws.autoscaling_group:
```

Comment:
Wouldn't it be clearer if a name was provided for each task?

Reply:
Better?

Reply:
Awesome! Thanks.